browser/Makefile.am

```diff
@@ -395,7 +395,7 @@ COOL_JS_LST =\
 	src/control/Control.NotebookbarDraw.js \
 	src/control/Control.NotebookbarBuilder.js \
 	src/control/Control.Layers.js \
-	src/control/Control.Sidebar.js \
+	src/control/Control.Sidebar.ts \
```

Comment:
There should be second list of files, please add that one too.
(It is  used for scanning for a translations)